client/logging/src/lib.rs

```diff
@@ -16,13 +16,19 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-//! TODO doc
+//! Substrate logging library.
+//!
+//! This crate uses tokio's [tracing](https://github.com/tokio-rs/tracing/) library for logging.
 
 #![warn(missing_docs)]
 
 mod event_format;
 mod layers;
 
+pub use sc_logging_proc_macro::*;
+#[doc(hidden)]
+pub use tracing;
```

Comment:
Is there any alternative to this? I'm guessing it's for the proc macro?

Reply:
No alternative sorry. Yes it's for the proc macro.